src/main/java/algorithms/orthogonalRangeSearching/twoDim/README.md

```diff
@@ -0,0 +1,68 @@
+# 2D Orthogonal Range Searching
+
+2D orthogonal range searching is a computational problem that involves efficiently answering queries about points 
+in a two-dimensional space that fall within a specified axis-aligned rectangular range (orthogonal range). In other 
+words, given a set of points in 2D, find me all the points that lie within a specific rectangle. 
+
+To do so, we can extend the idea of 1D range trees to the 2D context. 
+
+Strategy
+- Each node in the x-tree has a set of points in its subtree
+- Store a y-tree at each x-node containing all the points in the x-subtree
+
+![2DORS](../../../../../../docs/assets/images/2DORS.jpg)
+
+1. Build an x-tree using only x-coordinates. 
+   - This should be done in the exact same way as you would for a 1D range tree. 
+2. For every node in the x-tree, build a y-tree out of nodes in the x-subtree using only y-coordinates. 
+
+![2DORSTrees](../../../../../../docs/assets/images/2DORSTrees.jpg)
+
+Given the 2D range tree, we now want to query the points in a given rectangle 
+i.e. search(tree, xLow, xHigh, yLow, yHigh). 
+
+We first want to find the points that will satisfy the x-condition i.e. find me all points whose x-coordinates are 
+between xLow and xHigh. To do so, we first need to find our split node in the x-tree, by performing binary search while 
```

Comment:
ive seen you mentioned binary search a couple of times. You simply meant the standard findSplitNode function right? In a sense it is binary search - a balanced tree and roughly removing half of the nodes each time. Though this isn't the explicit binary search formulation we are most familiar with.

Not sure if it might cause confusion to others. Up to you to see fit if should add a quick expln on what you mean or use another term!